paper/bmd2023p.cls

```diff
@@ -123,7 +123,7 @@
 		\renewcommand{\do}[1]{\listsep~##1}%
 		\dolistloop\authorlist}}
 \newcommand{\addauthor}[2]{
-	\listadd{\authorlist}{#1~\textsuperscript{#2}}}
+	\listadd{\authorlist}{#1\textsuperscript{#2}}}

```

Comment:
My point is that we do not want to remove the tilde. Without the tilde we could have bad line breaks when the author list gets long.

Reply:
I don't get it. If we remove the tilde there is no space between the last character and the superscript anymore. I don't think latex would create a wrong line break there. Should be fine.

Reply:
The purpose of a `~` is to prevent line breaks. If the author list becomes long latex will decide where to break the lines when wrapping the author list to multiple lines. If you don't have a `~` then the line break could occur between the author's last name and the superscript. (unless the `\textsuperscript` "sticks" to the last name, maybe it does)

Reply:
I tried to find if `\textsuperscript` sticks to the word, but couldn't find it. I guess it is reasonable to believe that a superscript command would stick to the word. So I'll merge this.

Reply:
Thanks! 
To confirm that, I just experimented a bit with long names, removing individual characters till the line break jumped: The line break occurs either in front of the surname or after the comma that follows the superscript. Seems to be working!